From 4d652958bbbc4dc33b12d7dae47d92b380178ef0 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Mon, 20 Aug 2012 18:04:40 +0200
Subject: [PATCH 083/129] ALSA: hda - Add missing ifdef
 CONFIG_SND_HDA_POWER_SAVE to tracepoints

upstream: 099d53c308f50e8ee5b6638ec3f40f0104b3ee31

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: fupan li <fupan.li@windriver.com>
---
 sound/pci/hda/hda_trace.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/hda_trace.h b/sound/pci/hda/hda_trace.h
index 9a34b86..d42fe91 100644
--- a/sound/pci/hda/hda_trace.h
+++ b/sound/pci/hda/hda_trace.h
@@ -58,6 +58,7 @@ TRACE_EVENT(hda_bus_reset,
 	TP_printk("[%d]", __entry->card)
 );
 
+#ifdef CONFIG_SND_HDA_POWER_SAVE
 DECLARE_EVENT_CLASS(hda_power,
 
 	TP_PROTO(struct hda_codec *codec),
@@ -110,6 +111,7 @@ TRACE_EVENT(hda_power_count,
 		  __entry->card, __entry->addr, __entry->power_count,
 		  __entry->power_on, __entry->power_transition)
 );
+#endif /* CONFIG_SND_HDA_POWER_SAVE */
 
 TRACE_EVENT(hda_unsol_event,
 
-- 
1.7.5.4

